website/tutorial/deploying-a-kubernetes-application.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 4
+title: Deploying a Kubernetes application
+description: Deploying a Kubernetes application
+keywords: [podman desktop, podman, Kubernetes]
+tags: [podman-desktop, deploying-a-kubernetes-application]
+---
+
+# Deploying a Kubernetes application
+
+This tutorial covers the following end-to-end tasks required to deploy an application in a Kubernetes cluster:
+
+- Set the Kubernetes context
+- Creating a deployment
+- Creating a service
+
+If you have multiple Kubernetes contexts, you must set the correct context in which you want to create your application resources. Within a Kubernetes cluster, you can access the application by its internal IP address. However, if you want to make your application accessible from an outside network, you must expose the pod containing your application as a Kubernetes service.
+
+## Before you begin
+
+- [Installed Podman Desktop application](/docs/installation).
+- [A Podman machine](/docs/podman/creating-a-podman-machine).
+- [A running Kubernetes cluster](/docs/kind/creating-a-kind-cluster).
+- A deployment YAML configuration for use. Create a `Deployment` file using the following code, if you do not have one on your machine:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx-container
+          image: nginx:latest
+          ports:
+            - containerPort: 8080
+```
+
+This YAML configuration creates a deployment running three NGINX pods.
+
+- A service YAML configuration for use. Create a `Service` file using the following code, if you do not have one on your machine:
```

Comment:
`A service YAML configuration for use.` remove that

Explain what Service is briefly? Service opens the deployment so it's accessible?